src/reducers/index.js

```diff
@@ -1,4 +1,4 @@
-import { LOAD_ACTIVITIES, REGISTER, LOGIN, LOGOUT } from "../actions";
+import { LOAD_ACTIVITIES, REGISTER, LOGIN, LOGOUT, TOGGLE } from "../actions";
```

Comment:
TOGGLE - I don't see this chain of functions being used anywhere and the component works fine, so I think it can be removed.

Reply:
removed